roles/satellite-clone/tasks/main.yml

```diff
@@ -197,7 +201,7 @@
   file:
     path: /var/opt/rh/rh-postgresql12/lib/pgsql/data/postgresql.conf
     state: absent
-  when: satellite_version in ["6.8"]
+  when: satellite_version in ["6.8", "6.9"]
```

Comment:
I wonder of we should check `satellite_version is version("6.8", ">=")` instead, to allow for 6.10 and whatever.

See https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#comparing-versions

Reply:
That's nice. wonder why it wasn't used in several places in this repo...

Reply:
If I'd tell you, I'd owe you drinks, so you can forget again ;)

Reply:
`version` was changed from `version_compare` in 2.5 so it seems like we can't use this option since users might be on 2.4 for older sats

Reply:
Satellite 6.5 used Ansible 2.6: https://access.redhat.com/documentation/en-us/red_hat_satellite/6.5/html/installing_satellite_server_from_a_connected_network/installing_satellite_server#configuring_repositories_satellite

We don't support anything older I believe.

Reply:
I think the main question is: is this supposed to work without the "correct" Ansible repo enabled? Keeping 2.4 compatibility seems insane to me.

CC @mccun934

Reply:
I've been following the README in this repo :-/

The docs you linked are incorrect for not specifying which ansible repo to enable, right?

Reply:
I honestly don't know. For normal Sat operation, you need the *matching* Ansible repo. But s-clone is special.

It does enable the "right" repos itself:
https://github.com/RedHatSatellite/satellite-clone/blob/f374e242911c5c41e9f6e65cb173530f0f948356/roles/satellite-clone/tasks/main.yml#L58

But actually it doesn't, as it doesn't (before this PR) as it didn't enable Ansible at all...

Reply:
Since I haven't added any _new_ checks of this nature I'd like to keep the current approach